spec/notifications/publishers/job_application_data_expiry_notification_spec.rb

```diff
@@ -23,48 +23,4 @@
                      .and include(vacancy_applications_link)
     end
   end
-
-  describe "#timestamp" do
```

Comment:
I have removed the timestamp tests because they will be very flaky. The flakiness is caused by the way that notifications are now created. An event is created, and then when deliver is called on the event the notifications are created in the background. This means that they are not created within the time travel block. I tried adding a sleep to the before block, the updating the created_at time to match the required assertions but the tests were still flaky and I don't think add enough value to warrant investing anymore time into them.